samples/system-test/automlTranslation.test.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint node/no-extraneous-require: off */
+/* eslint node/no-unsupported-features/es-syntax: off */
+
+'use strict';
+
+const test = require(`ava`);
+const tools = require(`@google-cloud/nodejs-repo-tools`);
+
+const cmdDataset = `node ./automl/automlTranslationDataset.js`;
+const cmdModel = `node ./automl/automlTranslationModel.js`;
+const cmdPredict = `node ./automl/automlTranslationPredict.js`;
+
+// const flowerTrainData = `gs://`;
+// const cwd = path.join(__dirname, `..`);
+
+const testDataSetName = `testDataSet`;
+const dummyDataSet = `dummyDataSet`;
+const testModelName = 'dummyModel';
+const sampleText = `./automl/resources/testInput.txt`;
+
+const donotdeleteModelId = `TRL188026453969732486`;
+//const donotdeleteDatasetId = `TRL2551826603472450019`;
+
+// skipped because it's been taking too long to delete datasets
+test.skip(`It should create a create, list, and delete a dataset`, async t => {
+  // Check to see that this dataset does not yet exist
+  let output = await tools.runAsync(`${cmdDataset} listDatasets`);
+  t.false(output.includes(testDataSetName));
+
+  // Create dataset
+  output = await tools.runAsync(
+    `${cmdDataset} createDataset -n "${testDataSetName}"`
+  );
+  const dataSetId = output
+    .split(`\n`)[1]
+    .split(`:`)[1]
+    .trim();
+  t.true(output.includes(`Dataset display name:  ${testDataSetName}`));
+
+  // delete dataset
+  output = await tools.runAsync(
+    `${cmdDataset} deleteDataset -i "${dataSetId}"`
+  );
+  t.true(output.includes(`Dataset deleted.`));
+});
+
+// we make too models running this test, see hard-coded workaround below
+test.skip(`It should create a dataset, import data, and start making a model`, async t => {
+  // Check to see that this dataset does not yet exist
+  let output = await tools.runAsync(`${cmdDataset} listDatasets`);
+  t.false(output.includes(dummyDataSet));
+
+  // Create dataset
+  output = await tools.runAsync(
+    `${cmdDataset} createDataset -n "${dummyDataSet}"`
+  );
+  const dataSetId = output
+    .split(`\n`)[1]
+    .split(`:`)[1]
+    .trim();
```

Comment:
Can we use a regex?

Apply this comment throughout your PR.

Reply:
I copied this from cannonicals here: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/translate/automl/dataset_test.py ln 40